README.md

```diff
@@ -10,7 +10,7 @@ See https://www.portfolio-performance.info for more details.
 
 ## Prerequisites
 
-* [Java 8](https://www.java.com)
+* [Java 8](https://www.java.com) or [Java 11](https://github.com/ojdkbuild/ojdkbuild)
```

Comment:
That is actually a very good point - with the latest changes, Java 11 is a prerequisite